e2e/BinaryCoStream/src/jazz.tsx

```diff
@@ -14,10 +13,19 @@ const Jazz = createJazzReactApp();
 export const { useAccount, useCoState } = Jazz;
 
 export function AuthAndJazz({ children }: { children: React.ReactNode }) {
-  const [ephemeralAuth] = useState(ephemeralCredentialsAuth())
+  const [auth, state] = useDemoAuth();
+
+  const signedUp = useRef(false);
+
+  useEffect(() => {
+    if (state.state === "ready" && !signedUp.current) {
+      state.signUp('Mister X');
+      signedUp.current = true;
+    }
+  }, [state.state])
```

Comment:
Had to revert to the DemoAuth here because with the `ephemeralCredentialsAuth` the app wasn't working.

The E2E tests on the CI were working even before this change, so I'm not sure if it was something related to my local environment.

Reply:
Will need to investigate this, maybe the account doesn't sync properly after creation with `ephemeralCredentialsAuth`